src/item.rs

```diff
@@ -69,13 +71,15 @@ pub struct Item {
     #[serde(rename = "command")]
     pub kind: ItemKind,
 
+    #[serde(default = "Vec::new")]
     pub mappers: Vec<Mapper>,
 }
 
 #[derive(Debug, Clone, Deserialize)]
 pub enum Mapper {
     Regex {
-        regex: String, // `regex::Regex` does not yet support Deserialize unfortunately
+        #[serde(with = "serde_regex")]
+        regex: ::regex::Regex,
```

Comment:
I actually dont know `serde_regex`, until now (in my own project) I've deserialized a String and compiled it to a Regex later on. Is this only a shorthand for that? Or is this maybe able to remove some `\` clutter from the config file?
If I could write
```
regex = "\d\.\d\d \d\.\d\d"
```
instead of
```
regex = "\\d\\.\\d\\d \\d\\.\\d\\d"
```
that would be quite an increase of readability.